dlang.org.ddoc

```diff
@@ -145,7 +145,7 @@ _=
 
 DBLOG_LATEST_AUTHOR=
 DBLOG_LATEST_DATE=
-DBLOG_LATEST_LINK=
+DBLOG_LATEST_LINK=https://dlang.org/blog
```

Comment:
This is required for builds with `DIFFABLE=0` and the `chmgen` URL checking:

```
Error while processing file: /home/circleci/dlang.org/web/index.html
object.Exception@chmgen.d(577): Empty URL
----------------
```

https://circleci.com/gh/dlang/dlang.org/487?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link